lib/fast_inserter/fast_inserter_base.rb

```diff
@@ -92,20 +99,33 @@ def fast_insert_group(group)
       end
     end
 
+    def existing_values_sql(group_of_values)
+      sql = "SELECT #{@variable_columns.join(', ')} FROM #{@table_name} WHERE #{existing_values_static_columns}"
+
+      if @options[:filter_existing_in_database]
```

Comment:
@jordondornbos Might be worth copying the deleted comment about DB performance to this method so people know why this isn't on by default